Scenes/Lobby/Lobby.gd

```diff
@@ -16,22 +16,29 @@ func initialize_lobby():
 		add_item(LobbyManager.players[id])
 
 func StartGameBtn_configure():
-	if !scene_tree.is_network_server():
-		$MarginContainer/StartGameBtn.disabled = true
+	$MarginContainer/StartGameBtn.disabled = true
 
 func add_item(player_name):
 	var new_label = PlayerLabel.instance()
 	labels_container.add_child(new_label)
 	new_label.set_text(player_name)
+	if scene_tree.is_network_server() and labels_container.get_child_count() > 1:
+		$MarginContainer/StartGameBtn.disabled = false
```

Comment:
As you said, maybe you should check number of connected players instead of labels?